src/js/BookReader.js

```diff
@@ -389,18 +389,20 @@ BookReader.prototype.init = function() {
   }.bind(this));
 
   if (this.protected) {
-    $(document).on('contextmenu dragstart', '.BRpagediv1up', function() {
+    $(document).on('contextmenu dragstart', '.BRpagecontainer1up', function() {
       return false;
     });
     $(document).on('contextmenu dragstart', '.BRpageimage', function() {
       return false;
     });
-    $(document).on('contextmenu dragstart', '.BRpagedivthumb', function() {
+    $(document).on('contextmenu dragstart', '.BRpagecontainerthumb', function() {
       return false;
     });
```

Comment:
This can all be deleted now. The context menu appears, but it doesn't give you options to download the image because of the screen.